src/@types/tar-js/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for [~THE LIBRARY NAME~] [~OPTIONAL VERSION NUMBER~]
+// Project: [~THE PROJECT NAME~]
+// Definitions by: [~YOUR NAME~] <[~A URL FOR YOU~]>
+
+/*~ This is the module template file for class modules.
+ *~ You should rename it to index.d.ts and place it in a folder with the same name as the module.
+ *~ For example, if you were writing a file for "super-greeter", this
+ *~ file should be 'super-greeter/index.d.ts'
+ */
+
+/*~ Note that ES6 modules cannot directly export class objects.
+ *~ This file should be imported using the CommonJS-style:
+ *~   import x = require('someLibrary');
+ *~
+ *~ Refer to the documentation to understand common
+ *~ workarounds for this limitation of ES6 modules.
```

Comment:
I assume you just copied this over from tar-js, so this is more a "huh" comment than anything else, but: ES6 modules can export class objects perfectly well 🤷‍♀️

Reply:
Yeah, I forgot cleanup there. This is from the TypeScript Docs: https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-class-d-ts.html